exercises/practice/raindrops/.meta/config.json

```
@@ -1,6 +1,14 @@
 {
   "blurb": "Convert a number to a string, the content of which depends on the number's factors.",
-  "authors": [],
+  "authors": [
+    "fwip"
+  ],
+  "contributors": [
+    "devkabiir",
+    "ErikSchierboom",
+    "jackhughesweb",
+    "Stargator"
+  ],
```

```suggestion
  "contributors": [
  ],
```

---

Adding me as a contributor for the addition of a missing README and the revision of the exercise

```suggestion
  "contributors": [
    "Stargator"
  ],
```